apps/functions-subscription/package.json

```diff
@@ -10,9 +10,7 @@
     "lint": "eslint --fix",
     "lint:check": "eslint",
     "generate": "rimraf ./src/generated/definitions/internal && shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal",
-    "start": "func start --javascript",
-    "test": "vitest run --passWithNoTests",
-    "test:coverage": "vitest run --coverage --passWithNoTests"
```

Comment:
**note**: I have removed `test`, and `test:coverage` here because `vitest` is configured and managed at root level.

Reply:
these scripts can be useful at workspace level, when you are working focused on a workspace and you want to cache the results with turbo

Reply:
That makes sense 👍 , I'm going to reintroduce them.

Reply:
Done ✅